cincinnati/src/plugins/internal/graph_builder/dkrv2_openshift_secondary_metadata_scraper/test_fixtures/graph-data-6420f7fbf3724e1e5e329ae8d1e2985973f60c14/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/app-sre/busybox as downloader
+FROM registry.access.redhat.com/ubi9/ubi as downloader
```

Comment:
any reason why we want to use ubi image here? 
ubi image contains lots of binaries and os files that makes it heavy. We are using the container here just to store and extract the graph data files. Lets go for an image will small size.

Reply:
Per the commit https://github.com/openshift/cincinnati/pull/978/commits/c9844c4895c5c96a0f0d00950faa82548995763f:

```
The previously used quay.io/app-sre/busybox base container image is not
easily accessible and requires authorization. Only the `tar` command is
needed from the base image.

Instead, use an accessible base container image
registry.access.redhat.com/ubi9/ubi, which contains the necessary
tooling, authorization is not needed, and is from a reputable source.

The base image is quite large for its purpose; however, the image is
only used as a builder image and its content are not part of the final
image.
```

I acknowledge that even using the image for the build stage is quite an overkill. I tend to use Red Hat maintained container images where possible in such repositories. I am happy to replace the image with, for example, https://hub.docker.com/_/busybox if you feel it is appropriate.

Reply:
Can we use `ubi-micro` (smallest) or `ubi-minimal`?

Reply:
We could use `minimal` and do a `microdnf install tar`

Reply:
First try is unsuccessful. I will try to troubleshoot the issue later. 
```
$ podman build . -t test
[1/2] STEP 1/5: FROM registry.access.redhat.com/ubi9-minimal AS downloader
[1/2] STEP 2/5: ADD https://github.com/openshift/cincinnati-graph-data/archive/e8692fe50ccbfa525cce340f781d56d5a1d4364d.tar.gz /graph-data.tar.gz
--> Using cache 2b68ec2233a468a1fa828e03eebe61292cf8e32d756ecd4056d397dcb5850ba4
--> 2b68ec2233a4
[1/2] STEP 3/5: RUN microdnf install -y tar
--> Using cache a7461e7316ec640e47a3579241ad5029f5a2163a9c526d253a236677828dd0d7
--> a7461e7316ec
[1/2] STEP 4/5: RUN mkdir -p /graph-data
--> Using cache 9e14b8cf439b68c4dd34694dac05716b3ff56c8eac861e3311900572c902534e
--> 9e14b8cf439b
[1/2] STEP 5/5: RUN tar xav -C /graph-data -f /graph-data.tar.gz --no-same-owner
tar (child): gzip: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
Error: building at STEP "RUN tar xav -C /graph-data -f /graph-data.tar.gz --no-same-owner": while running runtime: exit status 2
```

Reply:
Right, we'd need `microdnf install tar gzip`

Reply:
Oh, yeah :facepalm:

Reply:
The `registry.access.redhat.com/ubi9-minimal` is now used. I have also updated the summaries of commits. The `dkrv2_openshift_secondary_metadata_scraper` scopes were just too long.